<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveCsvWindow</class>
 <widget class="QWidget" name="SaveCsvWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaveCsvWindow</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save to CSV - Select the pose</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_lable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>pose : </string>
   </property>
  </widget>
  <widget class="QListWidget" name="List_pose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="FunLabel_selectedpose">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_frames">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_frames">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>default</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_countforsequence">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Count for Sequence</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_saveformat">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>save format : &lt;label Name&gt;_&lt;Name&gt;_&lt;Frames&gt;&lt;Date&gt;.csv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_adjust">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_start">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 255, 184);</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_stop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 186, 174);</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Count">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count</string>
   </property>
  </widget>
  <widget class="QLabel" name="FuncLabel_count">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
